plugins/tpm-eventlog/fu-tpm-eventlog-common.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-tpm-eventlog-common.h"
+
+const gchar *
+fu_tpm_eventlog_hash_to_string (FuTpmEventlogHashKind hash_kind)
+{
+	if (hash_kind == FU_TPM_EVENTLOG_HASH_KIND_SHA1)
+		return "SHA-1";
+	if (hash_kind == FU_TPM_EVENTLOG_HASH_KIND_SHA256)
+		return "SHA-256";
+	return NULL;
```

Comment:
384 and 512 should be here too

Reply:
Added. It's a shame tss2 doesn't provide that. I've opened a bug about both helpers: https://github.com/tpm2-software/tpm2-tss/issues/1567